gymnasium/envs/box2d/lunar_lander.py

```diff
@@ -179,6 +179,7 @@ class LunarLander(gym.Env, EzPickle):
      The recommended value for `turbulence_power` is between 0.0 and 2.0.
 
     ## Version History
+    - v3: Reset wind and turbulence offset (`C`) whenever the environment is reset to ensure statistical independence between consecutive episodes.
```

Comment:
Can you add a link to the related GitHub issue See, MuJoC, ant for example. https://gymnasium.farama.org/main/environments/mujoco/ant/#version-history

Reply:
Of course - its done